server/src/main/java/org/apache/druid/server/QueryLifecycle.java

```diff
@@ -361,6 +362,8 @@ public void emitLogsAndMetrics(
       final Map<String, Object> statsMap = new LinkedHashMap<>();
       statsMap.put("query/time", TimeUnit.NANOSECONDS.toMillis(queryTimeNs));
       statsMap.put("query/bytes", bytesWritten);
+      statsMap.put("query/rowsScanned", rowsScanned);
```

Comment:
will this automatically result in 2 new columns for rowsscanned and cpuconsumed in OpenSearch?

Reply:
Yes, it will start coming in opensearch. We would need to index the new columns to query on the fields. But we'll start seeing logs in opensearch once this is deployed

Reply:
we dont need to index it. The columns will have values in ints. But we  do want to be able to sort by these columns. Do you know how to add ability to sort by these?

Reply:
Yeah, I think even for that we need to index these columns, or opensearch would not detect it as a column

Reply:
ok - i'll let you send a separate PR to index these.